docs/api.md

```diff
@@ -1697,6 +1706,30 @@ const puppeteer = require('puppeteer');
 })();
 ```
 
+#### page.enableWebVitalsReporting(enabledMetrics)
+
+- `enabledMetrics` <?[Array]<[string]>> Enable Web Vitals reporting. Supported metrics are `'CLS'`, `'FCP'`, `'LCP'`, `'FID'`, and `'TTFB'`. If omitted, all metrics will be reported.
+- returns: <[Promise]>
+
+*** Note *** The metrics will be emitted on [page.webvitals](#pagewebvitals).
+
+```js
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  page.webvitals.on('LCP', ({value}) => {
+    console.log('LCP:', value);
+  })
```

Comment:
Instead of just `value`, this is an opportunity to show what the complete object looks like. Are there other properties than `value`? Also please double-check the code style (e.g. missing semicolon, periods after comments, etc.)